BaseFilter/sections/general_extensions.txt

```diff
@@ -272,6 +272,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/146786
+asianage.com#%#AG_onLoad(() => { var element = document.querySelector('.rightCol'); if ( typeof element.innerHTML !== 'undefined' ) { element.innerHTML = element.innerHTML.replaceAll('ADVERTISEMENT', ''); } });
```

Comment:
I think that it would be better to check if element exist, something like:
```adblock
asianage.com#%#AG_onLoad(() => { var element = document.querySelector('.rightCol'); if ( element ) { element.innerHTML = element.innerHTML.replaceAll('ADVERTISEMENT', ''); } });
```
otherwise there will be an error in console (if there will be not `.rightCol`), it happens for example here - `https://www.asianage.com/search`